tests/src/ViewModel/PersonalisedCoverDownloadTest.php

```diff
@@ -21,7 +24,31 @@ public function it_has_data()
                     'text' => 'foo',
                 ],
             ],
-            'buttonCollection' => [
+            'picture' => [
+                'fallback' => [
+                    'altText' => '',
+                    'defaultPath' => '/default/path',
+                ],
+            ],
+            'a4ListHeading' => [
+                'heading' => 'A4',
+                'headingId' => 'buttons-a4',
+            ],
+            'a4ButtonCollection' => [
+                'buttons' => [
+                    [
+                        'classes' => 'button--default',
```

Comment:
@nlisgo I think at the moment why when I try to use these in the personalised-covers project running locally, and why the buttons are larger and result in some overlap, is the `pattern-library` buttons have classes of `button personalised__button button__gtm-download-pdf-a4`, but they do not in this test case. I have an example of how it might be possible to refactor `Button` to accept arbitrary class names on my PR (https://github.com/elifesciences/patterns-php/pull/745) and we could discuss on issues or in a call if the path forward is unclear.

Reply:
The `personalised__button` class is needed on each of the buttons. There is CSS that relies on the classes being there. I will see if I can amend the `patterns-php` to introduce those classes automatically. The other classes are intended for google tag manager but I'm not sure that they are necessary so I will pick this up after liasing with @discodavey.

Reply:
Leave this with me. I will try and simplify this.

Reply:
At this moment you don't need to worry about setting any classes.

Reply:
Thanks, so if that's how I have it now, then when patterns are merged, it should all fall into place